pt/lessons/advanced/otp-distribution.md

```diff
@@ -0,0 +1,484 @@
+---
+version: 1.0.1
+title: Distribuição OTP
+---
+
+## Introdução à Distribuição
+Nós podemos executar nossas aplicações Elixir em um conjunto diferente de nós, distribuídos em um único servidor ou entre múltiplos servidores. Elixir permite que nos comuniquemos entre esses nós por meio de alguns mecanismos diferentes, os quais iremos destacar nessa lição.
```

Comment:
I believe that "nós" is the best translation for "nodes". However reading the text becomes a little confusion, specially if we are not previously familiar with the concept. It is very easy to confuse "nós" (node) with "nós" (we). 

Do we have other options for this situation? Or we should stick with "nós"?

WDYT @elainenaomi @elixirschool/portuguese ?

Reply:
Nice point! I totally agree with you!

My suggestions:
- replace "nodes" with "nós de processamento". We can also use "nós de computação" or "unidades de processamento", but I've seen these terms used more in the academic context

- use the term "nodes" with a reference for the translation just in the first occurrence - but I think it is not referenced as "nodes" in Portuguese material

- add a link for the meaning of "nós" in the context of distribution - for instance, for the [Wikipedia page](https://pt.wikipedia.org/wiki/Sistema_de_processamento_distribu%C3%ADdo)


For now, I'll update the file to use "nós de processamento" instead of just "nós" to avoid the confusion.
Let me know if you have another alternative for this term, please 🙂

Reply:
@elainenaomi I think just using "nós de processamento" will do the trick. 

My main concern was with sentences like "Nós podemos executar nossas aplicações Elixir em um conjunto diferente de nós" where is very difficult to understand the correct "nós".